web/templates/web/children-list.html

```diff
@@ -13,6 +12,43 @@
 {% endblock %}
 {% block content %}
     <script>
+        $(document).ready(function() {
+            $('.datepicker').datepicker({})
+            .on('change', function () {
+               var age = getAge(this);
+               if (document.getElementById('age_calc')) {
+                   document.getElementById('age_calc').innerHTML = `Age: ${age}`;
+               } else {
+                   var x = document.createElement("p");
+                   x.setAttribute("id", "age_calc");
+                   x.setAttribute("class", "age_format")
+                   var t = document.createTextNode(`Age: ${age}`);
+                   x.appendChild(t);
+                   document.getElementsByClassName("form-group")[1].insertBefore(x,document.getElementById("id_birthday"));
+               }
+           });
+        });
+        function getAge(dateValue) {
```

Comment:
@icereval Is it worth pulling in moment.js for this kind of thing? I know how much we all love leap seconds. Not sure if it's worth the extra kb of js.